src/features/saksoversikt/utils.ts

```diff
@@ -216,72 +216,24 @@ function statusForPersonligOppmøte(personligOppmøte: Nullable<PersonligOppmøt
 }
 
 const getSatsStatus = (b: Behandling) => {
-    const ektefelle = b.behandlingsinformasjon.ektefelle;
-
     //vi sjekker på behandlingsstatus fordi at man kan endre på vilkår etter sats-steget, som ikke resetter sats.
     if (b.status === Behandlingsstatus.OPPRETTET || b.status === Behandlingsstatus.VILKÅRSVURDERT_AVSLAG) {
         return VilkårVurderingStatus.IkkeVurdert;
     }
-    if (
-        b.behandlingsinformasjon.bosituasjon &&
-        ektefelle &&
-        isPerson(ektefelle) &&
-        ektefelle.alder &&
-        ektefelle.alder >= 67
-    ) {
-        return VilkårVurderingStatus.Ok;
-    }
 
-    if (
-        ektefelle &&
-        b.behandlingsinformasjon.bosituasjon &&
-        b.behandlingsinformasjon.bosituasjon.ektemakeEllerSamboerUførFlyktning !== null
-    ) {
+    if (erBosituasjonFullstendig(b.grunnlagsdataOgVilkårsvurderinger.bosituasjon[0])) {
```

Comment:
Hva skjer om man ikke har noe bosituasjondata? Evt. hva om man har flere enn en?

Reply:
Hvis man ikke har noe data, så er dem ikke kommet så langt. Hvis den derimot ikke er fullstendig, så har dem ikke tatt et valg på sats-steget (ikke riktig for over 67+). 

Per nå, så skal det alltid bare være én bosituasjon. Videre så kommer det flere, og da må vi fikse på disse

Reply:
Okay, skjønner. Kunne vi ha gjort ett eller annet nå for å letter huske på å oppdatere det når det kommer flere bosituasjoner? Sånn det er skrevet nå så må vi kanskje i verste fall huske på å "fritekstsøke" i kodebasen etter `bosituasjon` for så å fikse de casene.
Har ikke noe konkret forslag her altså; lurer bare på å om vi kan gjøre jobben for fremtidsversjonene av oss selv litt lettere :p

Reply:
Tenker da forresten på det at å hente ut element 0 vil fortsette å funke helt fint dersom det er flere elementer også, noe som kan gjøre at vi ender opp med å ikke merke at det egentlig er feil.

Reply:
Kan kanske ha något likt `hentBosituasjonsgrunnlag(g: Grunnlag) => { return grunnlag.bosituasjon[0]}`.

Då blir det enklare att fjerne den funksjonen sen når vi har støtte for flera bosituasjonsgrunnlag, och då blir det error på alla plasser där vi har hentet ut den så? 😊

Reply:
I backend har jeg prøvd å samle disse funksjonene i Grunnlag.ts som extension-functions. Så når vi bytter til en faktisk liste kommer man langt med å ta utgangspunkt i de :shrug: